.changeset/eight-rules-laugh.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/admin": major
+---
+
+Change type of props `openedIcon` and `closedIcon` to `React.JSXElementConstructor<SvgIconProps>`
```

Comment:
Please mention the component of which the props are being changed

Reply:
✅